packages/ui/src/kit/VRadio/VRadio.theme-default.scss

```diff
@@ -59,4 +63,7 @@
   border-color: var(--endpass-ui-color-error);
 }
 
-
+.v-radio.theme-default .tooltip-molecule-icon {
+  vertical-align: top;
```

Comment:
v-radio should not get access to molecule icon.

you need remove this styles and in `label-molecule.theme-default.scss`
for `.label-molecule.theme-default .label-molecule-tooltip {`
set `vertical-align: sub;`

after for `.label-molecule.theme-default .label-molecule-icon {`
remove `vertical-align: sub;`

make the same operations for `tooltip-molecule.theme-default.scss`

Reply:
no, because in radio buttons there are specific styles for tooltip icon, which are completely different from label styles

Reply:
I agree that we can't change child scope from parent directly. Wrap molecule and apply styles to wrapper (wrapper - layout, tooltip - pure component) or change molecule inside

Reply:
and better would be use flex instead of hacking with `sub`

Reply:
if you will use flex, you need change all labels and text usage.
that's why `sub` is using as default. 
this is not hack